frontend/src/static/js/v_summary.js

```diff
@@ -131,6 +131,7 @@ window.vSummary = {
     },
     filterTimeFrame() {
       this.getFiltered();
+      this.$root.$emit('updateFilterTimeFrame', this.filtered, this.filterTimeFrame);
```

Comment:
I am wondering if we can modify the `filterTimeFrame` property of `v_zoom` directly and watch on the `filterTimeFrame` of `v_zoom` instead.
The reason I have this question is that we do not use `$emit` and `$on` before in our code. Not sure this method is ideal or not.

@ongspxm Can you also give your suggestion?

Reply:
From what I learnt about Vue, you can pass data from parent to child component using `props` while the other way round is through `$emit`. But since `v_zoom` is not the child of `v_summary` nor the other way round, I think the only way of passing data is through the listening of a global `$root` event. But yeah I am not sure so it is good to hear from someone who is more experienced in Vue.

Reply:
Since @ongspxm  is not replying, let's go to next review rounds.